index.html

```diff
@@ -1785,95 +1787,267 @@ <h2>Additional
+  <a href="#3fullyTransparentBlack">fully transparent black</a> rectangle,
+  with width and height dimensions from the <span class="chunk">IHDR</span> chunk. </p>
+
+<p>The static image may be included
+  as the first frame of the animation
+  by the presence of a single <span class="chunk">fcTL</span> chunk before <span class="chunk">IDAT</span>.
+  Otherwise, the static image is not part of the animation. </p>
+
+<p>Subsequent frames are encoded in <span class="chunk">fdAT</span> chunks,
+  which have the same structure as <span class="chunk">IDAT</span> chunks,
+  except preceded by a <a href="#4Concepts.APNGSequence">sequence number</a>.
+  Information for each frame
+  about placement and rendering
+  is stored in <span class="chunk">fcTL</span> chunks.
+  The full layout of <span class="chunk">fdAT</span> and <span class="chunk">fcTL</span> chunks is
+  <a href="#animation-information">described below</a>. </p>
+
+<p>The boundaries of the entire animation
+  are specified by the width and height parameters
+  of the `IHDR` chunk,
+  regardless of whether the default image is part of the animation.
+  The default image should be appropriately padded
+  with <a href="#3fullyTransparentBlack">fully transparent black</a> pixels
+  if extra space will be needed for later frames. </p>
+
+<p>Each frame is identical for each play,
+  therefore it is safe for applications to cache the frames. </p>
+
+</section>
+
+<section id="4Concepts.APNGSequence">
+<h3>Sequence numbers</h3>
+
+<p>The <span class="chunk">fcTL</span> and <span class="chunk">fdAT</span> chunks
+  have a zero-based, 4 byte sequence number.
+  Both chunk types share the sequence.
+  The purpose of this number is to detect (and optionally correct)
+  sequence errors in an Animated PNG,
+  since the PNG specification does not impose ordering restrictions on ancillary chunks. </p>
+
+<p>The first <span class="chunk">fcTL</span> chunk shall contain sequence number 0,
+  and the sequence numbers in the remaining
+  <span class="chunk">fcTL</span> and <span class="chunk">fdAT</span> chunks
+  shall be in ascending order,
+  with no gaps or duplicates.
+</p>
 
-<!-- <ol start="1"> --><ol>
-<li><a href="#ihdr-image-header"><span class="chunk">IHDR</span></a>: image
-header, which is the first chunk in a PNG datastream.</li>
+<p>The tables below illustrates the use of sequence numbers
+  for images with more than one frame,
+  and more than one <span class="chunk">fdAT</span> chunk
+  for the second frame.
+  (<span class="chunk">IHDR</span> and
+  <span class="chunk">IEND</span> chunks
+  omitted in these tables, for clarity).
+</p>
 
-<li><a href="#plte-palette"><span class="chunk">PLTE</span></a>:
-palette table associated with indexed PNG images.</li>
+<table class="Regular" summary=
+"sequence numbers, if the static image is also the first frame">
+<caption><a id="table52"><b>Table 5.2 &mdash; If the static image is also the first frame</b></a></caption>
```

Comment:
Lets put the id="table52" on the table tag instead of the a tag. Then we can remove the a tag.

Reply:
SGTM